packages/core/useAsyncQueue/demo.vue

```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+import { useAsyncQueue } from '.'
+
+const p1 = () => {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve(1000)
+    }, 10)
+  })
+}
+
+const p2 = (result: number) => {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve(1000 + result)
+    }, 20)
+  })
+}
+
+const { activeIndex, result } = useAsyncQueue([p1, p2])
+
+</script>
+
+<template>
+  <div>
+    <note>activeIndex: {{ activeIndex }}</note>
+    <note>result: {{ result }}</note>
+    <button @click="execute()">
```

Comment:
Maybe I'm missing something, what exactly does `execute()` refer to here?

Reply:
Maybe i was wrong,sorry